app/dao/annual_billing_dao.py

```diff
@@ -102,7 +102,7 @@ def set_default_free_allowance_for_service(service, year_start=None):
                                 f"{default_free_sms_fragment_limits['other'][year_start]}")
         free_allowance = default_free_sms_fragment_limits['other'][year_start]
 
-    dao_create_or_update_annual_billing_for_year(
+    return dao_create_or_update_annual_billing_for_year(
```

Comment:
why do we start returning this? Do we use it somewhere? If we do, how did it work before?

Reply:
This was included in https://github.com/alphagov/notifications-api/commit/ae57521b3947a854ce8459b06c268e533676a5aa where I simplified the case when annual_billing is missing when trying to get annual_billing for a year. 
I'm not sure why I had to add it back.

Reply:
oh ok, weird :o